src/components/cells/cell-menu.jsx

```diff
@@ -83,8 +95,13 @@ export class CellMenuUnconnected extends React.Component {
 
 export function mapStateToProps(state, ownProps) {
   const { cellId } = ownProps
+  let { languages } = state
+  languages = Object.keys(languages).filter(language => language !== 'js')
   const { skipInRunAll } = getCellById(state.cells, cellId)
-  return { skipInRunAll }
+  return {
+    skipInRunAll,
+    languages,
```

Comment:
just noticed that we should add proptypes for `languages`. also, maybe change the var name to `userLanguages` (just a teeny bit more specific)